### ISSUE TITLE
Brave: Stuck on 0 passes

### ISSUE THREAD

**Issue by StormProductionsMusic** [privacypass/challenge-bypass-extension#400]
**Describe the bug**
I've just installed PrivacyPass on Brave, and it doesn't want to increase my passes, no mater how much challenges I resolve. I resolved 10 challenges already and it still shows 0 passes.

**How to reproduce**
1. Install PrivacyPass on Brave;
2. Resolve a challenge from a supported sie, and see whether your passes are increasing.

**Expected behavior**
Everytime I solve a challenge, the number of passes should increase and I could redeem them later.

**System (please complete the following information):**
 - OS: Windows
 - Cloudflare tokens or hCaptcha tokens? hCaptcha
 - Browser: Brave
 - Privacy Pass Version: 3.0.6

**Comment by ericpa06** [comment]
I can confirm this bug. It's happening the same thing, And it's not even the first time this extension totally breaks . They will fix eventually, but it might take a few weeks. Honestly, I few they should pay more attention to this project :\

**Comment by GunGunGun** [comment]
Yeah, because it doesn't get updated regularly it becomes less effective.

**Comment by armfazh** [comment]
An issue on the service has been identified. A major reshape of the extension is in progress.

**Comment by StormProductionsMusic** [comment]
> An issue on the service has been identified. A major reshape of the extension is in progress.

Are you part of the team? If so, any clue when this new reshape will be released?

**Comment by armfazh** [comment]
Unfortunately, it may take few weeks.

**Comment by thibmeu** [comment]
An update on the current state of the extension:

A library supporting the [latest IETF draft](https://datatracker.ietf.org/wg/privacypass/about/) has been released on [cloudflare/privacypass-ts](https://github.com/cloudflare/privacypass-ts), and work on the extension is ongoing. 

In the meantime, both Cloudflare and hCaptcha moved on from the original Privacy Pass protocol to the IETF version, in the form of supporting Private Attestation Tokens. This improved interoperability and modularity at the protocol level. In addition, it opened the protocol interface to use cases beyond challenge pass, which this extension focused on. I invite you to check the IETF document to understand how this impacts your use cases.

As progress continues, there should be more news about the extension in early 2024.

**Comment by thibmeu** [comment]
What feature are you looking for?

**Comment by CuteistCat** [comment]
the ietf standard

**Comment by thibmeu** [comment]
Which part of the standard would you like to use that's not implemented by Silk?
